apps/s_client.c

```diff
@@ -974,7 +974,7 @@ int s_client_main(int argc, char **argv)
     c_quiet = 0;
     c_debug = 0;
     c_showcerts = 0;
-    c_nbio = 0;
+    c_nbio = 1;
```

Comment:
So the -nbio option should be a no-op, and we should update the documentation accordingly.